### PR TITLE
feat: Expose Datastore Admin generated client

### PR DESCRIPTION
**Pull request by pcostell**

### CODE REVIEW

File: grpc-google-cloud-datastore-v1/src/main/java/com/google/datastore/v1/DatastoreGrpc.java [hunk 1]
```diff
@@ -0,0 +1,1053 @@
+/*
```
Comment (kolea2) [inline grpc-google-cloud-datastore-v1/src/main/java/com/google/datastore/v1/DatastoreGrpc.java:1]: I am surprised to see this get generated for only the admin client - is this correct?

File: .repo-metadata.json [hunk 2]
```diff
@@ -13,7 +13,5 @@
   "api_id": "datastore.googleapis.com",
   "library_type": "GAPIC_COMBO",
   "api_description": "is a fully managed, schemaless database for\nstoring non-relational data. Cloud Datastore automatically scales with\nyour users and supports ACID transactions, high availability of reads and\nwrites, strong consistency for reads and ancestor queries, and eventual\nconsistency for all other queries.",
-  "excluded_dependencies": "grpc-google-cloud-datastore-v1",
```
Comment (kolea2) [inline .repo-metadata.json:16]: same-ish question here, why are we no longer excluding this for admin-only?

File: grpc-google-cloud-datastore-v1/src/main/java/com/google/datastore/v1/DatastoreGrpc.java (hunk 1 above)
Comment (pcostell) [reply grpc-google-cloud-datastore-v1/src/main/java/com/google/datastore/v1/DatastoreGrpc.java]: Owl seemed to be trying to add the repo as a dependency, but I don't see any reason that is actually true. I may have just misconfigured the owl configuration during that run. Removed it, we'll see if owl tries to put it back.

**Comment by pcostell** [comment]
I think actually ready for review now @kolea2, thanks for your help!

File: pom.xml [hunk 3]
```diff
@@ -156,6 +156,26 @@
...
+        <version>2.5.2-SNAPSHOT</version><!-- {x-version-update:grpc-google-cloud-datastore-v1:current} -->
+      </dependency>
+      <dependency>
+        <groupId>com.google.api.grpc</groupId>
+        <artifactId>grpc-google-cloud-datastore-admin-v1</artifactId>
+        <version>2.5.2-SNAPSHOT</version><!-- {x-version-update:grpc-google-cloud-datastore-admin-v1:current} -->
+      </dependency>
+      <dependency>
+        <groupId>com.google.api.grpc</groupId>
+        <artifactId>grpc-google-location</artifactId>
```
Comment (kolea2) [inline pom.xml:176]: what's this? I think the version should also be something different?

**Review by kolea2 — COMMENTED** [review]
looks great, one comment/question. Thanks!

File: pom.xml (hunk 3 above)
Comment (pcostell) [reply pom.xml]: Definitely should not be a required dependency. Removed it.

**Review by kolea2 — APPROVED** [review]

**Comment by fkorotkov** [comment]
Can we do the same for the regular Datastore client? There is no `grpc-google-cloud-datastore-v1` unfortunetly.

**Pull request merged**